engine/flink/util/src/test/scala/pl/touk/nussknacker/engine/flink/util/exception/ConfigurableExceptionhandlerSpec.scala

```diff
@@ -21,7 +21,7 @@ class ConfigurableExceptionhandlerSpec extends FunSuite with Matchers {
     //it's difficult to mock RuntimeContext for metrics so we switch it off..
     "exceptionHandler.withRateMeter" -> false,
     "restartStrategy.default.strategy" -> "fixed-delay",
-    "restartStrategy.default.delay" -> 10,
+    "restartStrategy.default.delay" -> "10 ms",
```

Comment:
Numerical delay will still work? If not, it will be needed entry in migration guide

Reply:
Yes, numerical delay still will be working.